### PR TITLE
Django 1.9 compatible - add_to_builtins check and doc-update

### LINKED ISSUE #11: [django 1.9a1] add_to_builtins removed
In django 1.9a1 `add_to_builtins` [was removed](https://docs.djangoproject.com/en/dev/releases/1.9/#django-template-base-add-to-builtins-is-removed). You're now supposed to put this into `settings.TEMPLATES['OPTIONS']['builtins']`.

@nigma If there is any hope of it getting merged and released I may make a PR that introduces a code side check whether we're in django 1.9+ and add documentation on putting easy-pjax into template options.

------

Traceback:
```python-traceback
(project)kondou:project/ (master) $ ./manage.py shell
Traceback (most recent call last):
  File "/home/kondou/git/project/lib/python3.4/site-packages/easy_pjax/__init__.py", line 15, in <module>
    from django.template import add_to_builtins
ImportError: cannot import name 'add_to_builtins'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/kondou/git/project/lib/python3.4/site-packages/django/core/management/__init__.py", line 350, in execute_from_command_line
    utility.execute()
  File "/home/kondou/git/project/lib/python3.4/site-packages/django/core/management/__init__.py", line 324, in execute
    django.setup()
  File "/home/kondou/git/project/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/kondou/git/project/lib/python3.4/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/home/kondou/git/project/lib/python3.4/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/home/kondou/git/project/lib64/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/kondou/git/project/lib/python3.4/site-packages/easy_pjax/__init__.py", line 18, in <module>
    from django.template.base import add_to_builtins
ImportError: cannot import name 'add_to_builtins'

### PR DESCRIPTION
**Pull request by Kondou-ger**
Closes #11

@nigma

### CODE REVIEW

**Comment by avelis** [comment]
:+1:  To merge this in. Django 1.9 is a little ways off but can get here sooner rather than later.

File: easy_pjax/__init__.py [hunk 1]
```diff
@@ -14,7 +14,12 @@
...
-    from django.template.base import add_to_builtins
+    try:
+        # import path changed in 1.8
+        from django.template.base import add_to_builtins
+    except ImportError:
+        # No more add_to_builtins in 1.9+
+        pass
 
-add_to_builtins("easy_pjax.templatetags.pjax_tags")
+if "add_to_builtins" in vars():
```
Comment (kmmbvnr) [inline easy_pjax/__init__.py]: May be it's better just to keep current auto-add behavior and modify TEMPLATES[BACKEND][OPTIONS][builtins] here.

Comment (Kondou-ger) [inline easy_pjax/__init__.py]: And get
```
    from django.template.base import add_to_builtins
ImportError: cannot import name 'add_to_builtins'
```
?

The `add_to_builtins()` function is **gone** in django 1.9 – adding easy_pjax to `TEMPLATES[BACKEND][OPTIONS][builtins]` doesn't make it reappear.

Comment (Kondou-ger) [inline easy_pjax/__init__.py]: Yup. Adding ourselves to `settings.TEMPLATES[0]['OPTIONS']['builtins']` now.

Not sure about the hardcoded `0` though …

**Comment by avelis** [comment]
@nigma It would appear that Django 1.9 was released today.
https://www.djangoproject.com/weblog/2015/dec/01/django-19-released/

I am not super positive of the viability of this pull request but it would appear to meet the requirements for Django 1.9.

**Comment by Kondou-ger** [comment]
I've made a version 1.2.0.1 (https://github.com/Kondou-ger/django-easy-pjax/commit/b0ef0070de49afa5cb98fbdab37e871aac8fde95) that includes my fix.  
If anyone wants to use it until this gets merged do something like this in your requirements.txt
```
# django-easy-pjax==1.2.0
# django-easy-pjax is broken with 1.9, here's a version that works with 1.9
git+https://github.com/Kondou-ger/django-easy-pjax.git@b0ef0070de49afa5cb98fbdab37e871aac8fde95
```

**Comment by Kondou-ger** [comment]
Closing this since @nigma prefers/merged the explicit version of this PR in contrast to the old auto add behaviour.

I will delete my branch/version 1.2.0.1 eventually (in a week or month or so, i don't know yet), too.

**Pull request closed without merging**